js/index.js

```diff
@@ -0,0 +1,61 @@
+//variables
+ $(".button-collapse").sideNav();
+$addMe = $('#left').children().children().children().children('.card');
```

Comment:
This would have been better if you added a class directly to what you were wanting to select.